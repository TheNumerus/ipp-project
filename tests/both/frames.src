.IPPcode20
CREATEFRAME
DEFVAR TF@_doot
MOVE TF@_doot string@Hello\032from\032temp\032frame!\010
WRITE TF@_doot
PUSHFRAME
DEFVAR LF@_local
MOVE LF@_local string@Now\032from\032<local>\032frame\010
MOVE LF@_doot LF@_local
WRITE LF@_doot
DEFVAR GF@_global
MOVE GF@_global string@Global\032also\032works\010
WRITE GF@_global
# why not pop that
POPFRAME