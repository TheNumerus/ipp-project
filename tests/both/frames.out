Hello from temp frame!
Now from <local> frame
Global also works
